reference/commands/creator/export-pkg.rst

```diff
@@ -123,69 +146,18 @@ with the ``export-pkg`` command. Check this :ref:`How to package existing binari
       build/lib/mylib.tmp
       build/file.obj
 
-  And assuming the ``Hello/0.1@user/stable`` recipe has a ``package()`` method like this:
+  we would need a slightly more complicated *conanfile.py* than in the previous
+  example to select which files to copy, we need to change the patterns in the
+  ``package()`` method:
 
   .. code-block:: python
 
       def package(self):
-          self.copy("*.h", dst="include", src="include")
-          self.copy("*.lib", dst="lib", keep_path=False)
+         self.copy("*.h", dst="include", src="include")
+         self.copy("*.lib", dst="lib", keep_path=False)
 
-  Then, the following code will create a package in the conan local cache:
+  Now, we can run Conan to create the package:
 
   .. code-block:: bash
 
-      $ conan export-pkg . Hello/0.1@user/stable -pr=myprofile --source-folder=sources --build-folder=build
-
-  And such package will contain just the files:
-
-  .. code-block:: text
-
-      include/mylib.h
-      lib/mylib.lib
-
-- Building a conan package (for architecture x86) in a local directory and then send it to the local cache:
```

Comment:
I am ok with removing this part from this section, but I would like to see it as a more thorough summary in the https://docs.conan.io/en/latest/developing_packages/package_dev_flow.html section, in which the whole step-by-step commands of local development are shown, all together and using the working recipe from the repo (should be moved to examples), starting with a ``git clone examples``

Reply:
Yes, I agree we need to revisit that section now that @danimtb has merged the local workflow example into the `examples` repo https://github.com/conan-io/examples/pull/12, but I would do it in a different issue.